client/js/components/user/DpUserList/DpUserList.vue

```diff
@@ -223,17 +223,22 @@ export default {
     }, 500),
 
     getItemsByPage (page) {
-      const search = {
-        value: this.searchValue
-      }
       this.isLoading = true
       page = page || this.currentPage
 
       this.userList({
         page: {
           number: page ?? 1
         },
-        search: (this.searchValue !== '') ? search : {},
+        filter: {
+          firstnameFilter: {
+            condition: {
+              path: 'firstname',
+              operator: 'STRING_CONTAINS_CASE_INSENSITIVE',
+              value: this.searchValue
+            }
```

Comment:
I think something like that
```suggestion
              operator: 'STRING_CONTAINS_CASE_INSENSITIVE',
              value: this.searchValue,
              memberOf: 'name'
            },
            lastnameFilter: {
            condition: {
              path: 'firstname',
             operator: 'STRING_CONTAINS_CASE_INSENSITIVE',
              value: this.searchValue,
            memberOf: 'name'
            },
            name: {
              group: {
                conjunction: OR
              }
            }
```

Reply:
thank you, done here --> https://github.com/demos-europe/demosplan-core/pull/3360/commits/7fd0b95adf7c85f92c8465972e9eea909ed03c3b